pos_invoice_confirm/models/pos_config.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2019 Druidoo - Iván Todorovich
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    iface_invoice_order_reminder = fields.Boolean(
+        'Show invoice order reminder')
```

Comment:
```suggestion
    iface_invoice_order_confirm = fields.Boolean(
        'Invoice reminder',
        help=(
            'If the order is not set to be invoiced, and the user '
            'tries to validate it, it\'ll show a confirmation popup '
            'asking the user if he wants to invoice it or not.\n\n'
            'This popup will not be shown if the order doesn\'t '
            'have a customer set.'
        ),
    )
```

Let's rename this field to `iface_invoice_order_confirm`, as it's more coherent with the module name.
Can you change it on the other files?